b2/console_tool.py

```diff
@@ -1055,7 +1099,10 @@ def decode_sys_argv():
 
 def main():
     info = SqliteAccountInfo()
-    b2_api = B2Api(info, AuthInfoCache(info))
+    b2_http = B2Http()
```

Comment:
This is a great idea. Maybe lets call it `clock_antiskew_hook` though? `clock_skew_hook` sounds like something that has a purpose of making sure the clock is properly skewed...